packages/@react-spectrum/s2/stories/Menu.stories.tsx

```diff
@@ -482,32 +491,64 @@ export const SelectionGroups = (args) => {
   let [group2, setGroup2] = useState<Selection>(new Set());
   return (
     <MenuTrigger {...triggerProps}>
-      <Button aria-label="Actions for selected resource"><NewIcon /></Button>
+      <Button aria-label="Text actions"><TextIcon /></Button>
       <Menu {...menuProps}>
         <MenuSection>
           <Header>
-            <Heading>Actions</Heading>
+            <Heading>Clipboard</Heading>
           </Header>
-          <MenuItem>Action 1</MenuItem>
-          <MenuItem>Action 2</MenuItem>
+          <MenuItem>
+            <Cut />
+            <Text slot="label">Cut</Text>
+          </MenuItem>
+          <MenuItem>
+            <Copy />
+            <Text slot="label">Copy</Text>
+          </MenuItem>
+          <MenuItem>
+            <Paste />
+            <Text slot="label">Paste</Text>
+          </MenuItem>
         </MenuSection>
         <MenuSection selectionMode="single" selectedKeys={group1} onSelectionChange={setGroup1}>
           <Header>
-            <Heading>Single-select</Heading>
+            <Heading>Text Alignment</Heading>
           </Header>
-          <MenuItem id={1}>One</MenuItem>
-          <MenuItem id={2}>Two</MenuItem>
-          <MenuItem id={3}>Three</MenuItem>
+          <MenuItem id={1}>
+            <AlignLeft />
+            <Text slot="label">Left</Text>
+          </MenuItem>
+          <MenuItem id={2}>
+            <AlignMiddle />
+            <Text slot="label">Center</Text>
+          </MenuItem>
+          <MenuItem id={3}>
+            <AlignRight />
+            <Text slot="label">Right</Text>
+          </MenuItem>
         </MenuSection>
         <MenuSection selectionMode="multiple" selectedKeys={group2} onSelectionChange={setGroup2}>
           <Header>
-            <Heading>Multi-select</Heading>
+            <Heading>Font Style</Heading>
           </Header>
-          <MenuItem id={4}>Four</MenuItem>
-          <MenuItem id={5}>Five</MenuItem>
-          <MenuItem id={6}>Siz</MenuItem>
+          <MenuItem id={4}>
+            <Bold />
+            <Text slot="label">Bold</Text>
+          </MenuItem>
+          <MenuItem id={5}>
+            <Italic />
+            <Text slot="label">Italic</Text>
+          </MenuItem>
+          <MenuItem id={6}>
+            <Underline />
+            <Text slot="label">Underline</Text>
+          </MenuItem>
         </MenuSection>
       </Menu>
     </MenuTrigger>
   );
 };
+
+SelectionGroups.parameters = {
```

Comment:
just an fyi, this is a deprecated syntax for whenever we go to storybook 8
you can make a story
```
const SelectionGroups = {
  render: (args) => <SelectionGroupsOriginalFunction {...args} />,
  parameters...
}
```

but they don't like it when you make a function the story and then set parameters on that function object

Reply:
that syntax is unfortunate because it breaks react fast refresh (story exports are no longer valid react components). 😢